crypto/fipsmodule/dh/dh.c

```diff
@@ -415,6 +415,8 @@ int DH_compute_key_hashed(DH *dh, uint8_t *out, size_t *out_len,
 
 int DH_size(const DH *dh) { return BN_num_bytes(dh->p); }
 
+int DH_bits(const DH *dh) { return DH_num_bits(dh); }
```

Comment:
Is this always a safe cast, this could cast the number bits to a negative size right? I realize this is very unlikely since no one should (or even can?) make a 2^31 + 1 sized DH number.

Reply:
I just noticed this is following the same pattern as DH_size.